app/controllers/organizations/activations_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Organizations
+  class ActivationsController < Organizations::BaseController
+    def update_activation
+      @user = User.find(params[:activation_id])
```

Comment:
I think it would be more appropriate to send `user_id` in the params. We may need to explicitly pass that in the path helper. So, when you remove this custom action and just use the `update` action it will be something like `activations_path(user_id: user.id)` 
I think rails will yell at you for not having an activations id, but if we change the routes to `resource :activations`, instead of `resources` we should be fine. That make sense?